ui/preferences/src/main/res/xml/preferences_downloads.xml

```diff
@@ -47,6 +47,19 @@
                 android:key="prefMobileUpdateTypes"
                 android:summary="@string/pref_mobileUpdate_sum"
                 android:title="@string/pref_mobileUpdate_title"/>
+        <SwitchPreferenceCompat
+                android:defaultValue="false"
+                android:enabled="true"
+                android:key="prefNetworkConstraintsDisabled"
+                android:summary="@string/pref_networkConstraintsDisabled_sum"
+                android:title="@string/pref_networkConstraintsDisabled_title"/>
```

Comment:
This does not need to be a user-visible setting. It can be controlled just from within the "allow mobile download" popup window